src/views/headerView.jsx

```diff
@@ -211,7 +216,7 @@ export const ConferenceHeader = ({ onParticipantListOpen }) => {
         ]}
         centerComponents={[<SpeakerTag key={0} />]}
         rightComponents={[
-          <PIPComponent key={0} />,
+          localPeer.roleName !== HLS_VIEWER_ROLE && <PIPComponent key={0} />,
```

Comment:
it will be better to move this condition to a variable, we should remove the speaker tag as well(as it won't be real time).